src/assets/i18n/en.json5

```diff
@@ -2960,11 +2960,11 @@
 
   "submission.sections.describe.relationship-lookup.external-source.import-button-title.Journal Volume": "Import remote journal volume",
```

Comment:
If I hover over the import buttons for `PubMed` or `arXiv`, I see a missing key of `submission.sections.describe.relationship-lookup.external-source.import-button-title.isAuthorOfPublication`

Reply:
@tdonohue Added it in